Serverless-Functions/homework/responses/1.5-blog.md

```diff
@@ -0,0 +1,60 @@
+**Let’s wrap up what you learned and did in Week 1: downloading and using Visual Studio Code, learning to navigate GitHub and committing changes, and deploying a simple Azure function.**
+<br><br/>
+
+Visual Studio Code (or VS code) was downloaded from this link: https://code.visualstudio.com
+
+Inside VS code, you downloaded four extensions: Azure Account, Azure App Service, Azure Functions, and Live Server. All of the Azure extensions allow you to work on your Azure Function App in VS code instead of working directly through the Microsoft portal. Live Server is a quick and temporary testing server, and you can use it to test HTML pages.
```

Comment:
link resources so students can find everything easier. (ie. where can you download the extensions?)